hazardous/tests/test_ipcw.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+import pytest
+from lifelines.datasets import load_regression_dataset
+from numpy.testing import assert_array_almost_equal
+
+from .._ipcw import IpcwEstimator
+
+
+@pytest.mark.parametrize("competitive_risk", [True, False])
+def test_ipcw(competitive_risk):
+    X = load_regression_dataset()
+    y = dict(
+        event=X["E"],
+        duration=X["T"],
+    )
+    if competitive_risk:
+        coef = np.random.choice([1, 2], size=y["event"].shape[0])
```

Comment:
I think we should always seed RNGs in test to avoid random, rare, non-reproducible errors on the CI.

```suggestion
        rng = np.random.default_rng(0)
        coef = rng.choice([1, 2], size=y["event"].shape[0])
```

We could make the test non-rng sensitive by parameterizing it with a seed, e.g.:

```python
@pytest.mark.parametrize("competing_risks", [True, False])
@pytest.mark.parametrize("random_seed", range(3))
def test_ipcw(competing_risks, random_seed):
    ...
        rng = np.random.default_rng(random_seed)
        coef = rng.choice([1, 2], size=y["event"].shape[0])
    
```